language/documentation/book/src/address.md

```diff
@@ -6,17 +6,60 @@ Although an `address` is a 128 bit integer under the hood, Move addresses are in
 
 You can use runtime address values (values of type `address`) to access resources at that address. You *cannot* access modules at runtime via address values.
 
-## Literals
+## Addresses and Their Syntax
 
-`address` literals are 16-byte hex literals, i.e. `0x<hex encoded value>`. For convenience, leading `0`s are added for literals that are too short.
+Addresses come in two flavors, named or numerical. The syntax for a named address follows the
+same rules for any named identifier in Move. The syntax of a numerical address is not restricted
+to hex-encoded values, and any valid [`u128` numerical value](./integers.md) can be used as an
+address value, e.g., `42`, `0xCAFE`, and `2021` are all valid numerical address
+literals.
+
+To distinguish when an address is being used in an expression context or not, the
+syntax when using an address will differ slightly depending on the context it's
+being used in:
+* When an address is used as an expression the address must be prefixed by the `@` character, i.e., `@`[`<numeric_value>`](./integers.md)` or `@<named_address_identifier>`.
+* Outside of expression contexts, the address may be written without the leading `@` character, i.e., [`<numeric_value>`](./integers.md) or `<named_address_identifier>`.
+
+In general, you can think of `@` as an operator that takes an address from being a namespace item to being an expression item.
+
+## Named Addresses
+
+As mentioned above, Move supports the concept of named addresses. Named
```

Comment:
Kill this first sentence. It was already mentioned above, and this sentence adds no value.